ecosystem/sep-0012.md

```diff
@@ -302,6 +340,41 @@ For example:
 }
 ```
 
+## Customer PUT Verification
+
+This endpoint allows servers to accept data values, usually confirmation codes, that verify a previously provided field via `PUT /customer`, such as `mobile_number` or `email_address`. Note that while fields such as `photo_proof_residence` or `notary_approval_of_photo_id` are verifications of other fields described in [SEP-9](#sep-0009.md), the server does not require the associated fields _before_ verification can be accomplished.
```

Comment:
~❓ Where are `photo_proof_residence` and `notary_approval_of_photo_id`, and `mobile_number_verification` used in the example, defined?~

💡 I think it is worth moving the paragraph about how the `_verification` fields are defined, that currently lives in the request section, ahead of any examples of fields here. When reading top-to-bottom seeing examples ahead of a definition creates confusion.

Reply:
I found it difficult to move the verification text above the examples from a flow perspective, so I moved the relevant example down into the request section.